benchtop/src/cli.rs

```diff
@@ -63,21 +33,48 @@ impl Display for Backend {
     }
 }
 
+/// Parameters to the init command.
+#[derive(Debug, Args)]
+pub struct InitParams {
+    #[clap(flatten)]
+    pub workload: WorkloadParams,
+
+    /// The backend to run the workload against.
+    #[arg(required = true, long, short)]
+    pub backend: Backend,
+}
+
+/// Parameters to the run command.
+#[derive(Debug, Args)]
+pub struct RunParams {
+    #[clap(flatten)]
+    pub workload: WorkloadParams,
+
+    #[clap(flatten)]
+    pub limits: RunLimits,
+
+    /// The backend to run the workload against.
+    #[arg(required = true, long, short)]
+    pub backend: Backend,
+
+    /// Whether to reset the database.
+    #[clap(default_value = "true")]
+    #[arg(long, short)]
+    pub no_reset: bool,
```

Comment:
Wouldn't it be easier to have a `reset` flag defaulting to false? 
This way, logically adding the flag `--reset` would trigger a database reset

Reply:
yeah, true.